README.md

```diff
@@ -42,11 +42,14 @@ At last, visit `localhost:3001`.
 To run locally in production mode run
 
     $ npm run dist
-    $ NODE_ENV=production APP_ENV=[environment variable] npm start
+    $ clientID=[client id] clientSecret=[client secret] NODE_ENV=production APP_ENV=[environment variable] npm start
 
 and visit `localhost:3001`.
 NOTE: Currently the only working `environment variable` is `development`.
 
+## Client Id and Secret
```

Comment:
Instead of saying coworker, can we say NYPL digital department, or any other name to indicate our team, so that once this application is used or introduced to other team or depts, they can get a more clear idea. Maybe contacts too if not too much.